contrib/PKGBUILD

```diff
@@ -0,0 +1,35 @@
+# Maintainer: Bruno Pagani (a.k.a. ArchangeGabriel) <archange@archlinux.org>
+# Contributor: Mirco Tischler <mt-ml at gmx dot de>
+
+pkgname=fwupd
+pkgver=LATEST
```

Comment:
If artifacts were to be saved, it would only be on tagged releases (eg the next one would be 0.9.5).

The main reason to consider saving artifacts would be for drive by downloaders who go git the releases page and want to grab the latest release there if they're not in the packaging system yet.  

I guess this is up to you on how you'd like to approach it.  If you anticipate people using the docker container and script to do git builds and want to know what commit they're at etc.